### PR TITLE
Add Holybro PX4 vision v1.5 Airframe

### PR DESCRIPTION
**Pull request by vincentpoont2**
Add Holybro PX4 vision v1.5 Airframe so user can select it in QGC.
Holybro PX4 Vision v1.5 has some differences from the v1 would require different parameter.

image

[More info regarding PX4 Vision v1.5 ](https://shop.holybro.com/px4-vision-dev-kit-v15_p1342.html)

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/airframes/4020_holybro_px4vision_v1_5 [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+# @class Copter
+#
+# @board px4_fmu-v2 exclude
+# @board bitcraze_crazyflie exclude
+#
+
+. ${R}etc/init.d/rc.mc_defaults
+
+# Commander Parameters
+param set-default COM_OBS_AVOID 1
```
Comment (Jaeyoung-Lim) [inline ROMFS/px4fmu_common/init.d/airframes/4020_holybro_px4vision_v1_5:15]: Just curious, does this still work with the pixhawk 6c?

Comment (vincentpoont2) [reply ROMFS/px4fmu_common/init.d/airframes/4020_holybro_px4vision_v1_5:15]: We tested it in the factory, do you see an issue here? @Jaeyoung-Lim

**Comment by julianoes** [comment]
Rebased and force pushed.

**Comment by vincentpoont2** [comment]
@julianoes Please merge and backport (if possible). I tested this and it looks good on myside.

**Review by dagar — APPROVED** [review]

**Pull request merged**